test/studies/cholesky/jglewis/version2/elemental/scalar_inner_product_cholesky.chpl

```diff
@@ -83,7 +83,7 @@ module scalar_inner_product_cholesky  {
 	A (j, j)      = sqrt ( A (j, j) );
 	A (j+1.., j ) = A (j+1.., j) / A (j, j);
       }
-      else 
+      else
```

Comment:
In these cholesky tests, whitespace changes also outshadow the PR-related changes. It would be nicer without them.